docs/source/tertiary/whole-genome-regression.rst

```diff
@@ -181,23 +179,40 @@ Example
 
 1. Initialization
 =================
 
-When the ``RidgeReducer`` is initialized, it assigns names to the provided alphas and stores them in a python dictionary accessible as ``RidgeReducer.alphas``. If alpha values are not provided, they will be generated during ``RidgeReducer.fit`` based on the number of unique headers in the blocked genotype matrix :math:`X_0`, denoted by :math:`h_0`, and a set of heritability values. More specifically,
+The ``RidgeReduction`` is initialized by receiving the following parameters:
+
+Parameters
+----------
+
+- ``block_df``: Spark DataFrame representing the beginning block matrix
+- ``label_df``: Pandas DataFrame containing the target labels used in fitting the ridge models
+- ``sample_blocks``: Dictionary containing a mapping of sample block IDs to a list of corresponding sample IDs
+- ``covariate_df``: Pandas DataFrame containing covariates to be included in every model in the stacking
+  ensemble (optional)
+- ``add_intercept``: Whether an intercept column (all ones) should be added to the covariates (as the first column) (optional, default = `True`).
+- ``alphas``: a numpy array of alpha values used in the ridge reduction (optional). If not provided GloWGR generates them.
+- ``label_type``: String to determine type treatment of labels (optional). It can be ``'detect'`` (default), ``'binary'``, or ``'quantitative'``.
+
+Notice that the user can specify that an intercept column is added to the covariates matrix or not.
+
+If alpha values are not provided, they will be generated based on the number of unique headers in the blocked genotype matrix :math:`X_0`, denoted by :math:`h_0`, and a set of heritability values. More specifically,
 
 .. math::
 
     \alpha = h_0\big[\frac{1}{0.99}, \frac{1}{0.75}, \frac{1}{0.50}, \frac{1}{0.25}, \frac{1}{0.01}\big]
 
-These values are only sensible if the phenotypes are on the scale of one.
+
+Moreover, ``RidgeReduction`` has the ability to detect whether it should treat phenotypes as quantitative or binary. If ``label_type='detect'``, the problem will be treated as binary-phenotype problem if all phenotypes vectors are binary. Otherwise, the problem will be treated as a quantitative-phenotype problem. The user can force the type of the phenotypes by changing the value of ``label_type`` to ``'binary'`` or ``'quantitative'``. Note that forcing quantitative phenotype values to be treated as binary will throws an error.
```

Comment:
Wording nit: `the problem will be treated as binary-phenotype problem` -> `the problem will be treated as a binary-phenotype problem`

Reply:
Wording nit: `will throws an error` -> `will cause an error to be thrown`